msg/ExternalAdc.msg

```diff
@@ -0,0 +1,6 @@
+uint64 timestamp		# time since system start (microseconds)
```

Comment:
Can you do this using the existing msg (AdcReport.msg)?

I would use the same msg, but a custom topic. You do this by adding `# TOPICS external_adc` at the bottom of AdcReport.msg. 

Example https://github.com/PX4/PX4-Autopilot/blob/b632a2750d7855cc7932948a59e1bf2f158d3a62/msg/VehicleGlobalPosition.msg#L30

Reply:
I have added topic in the existing msg (AdcReport.msg). Please check and let me know if any other changes are required.